src/styles/modules/modal-issue.less

```diff
@@ -1,10 +1,24 @@
 .modal-issue {
+    margin: 0 auto;
     padding: 10rem;
+    height: auto;
```

Comment:
This line prevents me from scrolling an issue should its content take up more height than the viewport.